econia-labs/econia
# [ECO-486][ECO-538] Add balance update support

* Delete deprecated Rust README
* Add processor for `MarketAccounts` table handles
* Add missing postgres views to old migrations
* Add processor for `MarketAccount` table entries
* Add view function to join on table handle for balance updates

## To verify

```yaml
health_check_port: 8085
server_config:
  processor_config:
    type: econia_transaction_processor
    econia_address: 0xc0de11113b427d35ece1d8991865a941c0578b0f349acabbe9753863c24109ff
  postgres_connection_string: postgres://econia:econia@postgres:5432/econia
  indexer_grpc_data_service_address: https://grpc.testnet.aptoslabs.com:443
  indexer_grpc_http2_ping_interval_in_secs: 60
  indexer_grpc_http2_ping_timeout_in_secs: 10
  auth_token: <my_auth_token>
  starting_version: 683453241
```

Run Docker compose instructions per the docs site then see http://0.0.0.0:3001/balance_updates
